src/app/protocol-handler.ts

```diff
@@ -61,7 +61,7 @@ class ProtocolHandler {
      */
     public processArgv(argv?: string[]): void {
         const protocolUriFromArgv = getCommandLineArgs(argv || process.argv, protocol.SymphonyProtocol, false);
-        if (isWindowsOS && protocolUriFromArgv) {
+        if (protocolUriFromArgv) {
```

Comment:
reason for this change please?

Reply:
We need to handle `protocol handler` from the cmd line for both `Windows` & `Mac` so removed this check